jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -303,7 +304,7 @@
                                     (send self :get-proximity l/r :left))
                                  (send self :interpolatingp))
                        (unix::usleep 1000))))
-                 (send self :cancel-move-hand)
+                 (send self :cancel-move-hand l/r)
                  (send self :cancel-angle-vector)
```

Comment:
I think it becomes very big diffs.
IMO, current style is simple and follows hand handling of other packages such as fetcheus and naoqieus.
What do you think?